new-pipeline/src/dist.js

```diff
@@ -143,10 +162,14 @@ $(function () {
 
       $("#channel-version")
         .change(function () {
-          updateOptions(function () {
-            $("#measure")
-              .trigger("change");
-          });
+          changePending = true;
+          gFilterChangeTimeout = setTimeout(function () {
```

Comment:
We could factor out this common pattern and reuse it in the three places we need it:

```js
function setFilterChangeTimeout(timeoutFunction, timeoutDelay) {
  gChangePending = true;
  gFilterChangeTimeout = setTimeout(() => {
    timeoutFunction();
    changePending = false;
  }, timeoutDelay);
}
```

(the place to put it would be at the bottom)

Reply:
Sure, I'll do this too